src/Symfony/Component/DependencyInjection/Loader/FileLoader.php

```diff
@@ -139,13 +144,25 @@ private function findClasses($namespace, $pattern, $excludePattern)
             if (!preg_match('/^[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+(?:\\\\[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*+)*+$/', $class)) {
                 continue;
             }
+
+            try {
+                $r = $this->container->getReflectionClass($class);
+            } catch (\ReflectionException $e) {
+                $classes[$class] = sprintf(
+                    'While discovering services from namespace "%s", an error was thrown when processing the class "%s": %s',
```

Comment:
nit pick :). I think we usually try to end errors with a period. So the end of this could be:

> ... processing the class "%s": "%s".

Reply:
✅